### DISCUSSION TITLE
How to use Drawable as title?

### DISCUSSION THREAD

**Discussion by anta40** [AppIntro/AppIntro#1086] [Q&A] [answered]
I'm using AppIntro 6.2.0 on my Android Java project (Android Studio Flamingo)

![aio](https://user-images.githubusercontent.com/16976/234571220-fb29f1f9-22b0-484d-939a-b65cd39a0fc4.jpg)

```
  addSlide(AppIntroFragment.createInstance(
                "Howdy",
                "Metode pembayaran tanpa batas tak ada peluang kandas",
                R.drawable.ic_slide2,
                R.color.colorPrimaryLight,
                R.color.colorPrimaryLight,
                R.color.colorPrimaryLight,
                R.font.open_sans_regular,
                R.font.open_sans_regular,
                R.drawable.app_background
        ));
```
Almost got the screen right. Is it possible to replace "Howdy" (a text) with another Drawable?

**Comment by cortinico** [comment, marked as answer]
> Almost got the screen right. Is it possible to replace "Howdy" (a text) with another Drawable?

Nope that's not possible. You can either:
- Provide a single Drawable that contains the title
- Use a custom layout for the specific slide
